website/www/site/content/en/blog/beam-2.58.0.md

```diff
@@ -0,0 +1,130 @@
+---
+title:  "Apache Beam 2.58.0"
+date:   2024-07-22 13:00:00 -0800
+categories:
+  - blog
+  - release
+authors:
+  - jrmccluskey
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+We are happy to present the new 2.58.0 release of Beam.
+This release includes both improvements and new functionality.
+See the [download page](/get-started/downloads/#2570-2024-07-22) for this release.
```

Comment:
I assume this URL will be updated before we publish.

Reply:
Yes the dates are going to be updated now that the RC has been approved